jobs/build.jpl

```diff
@@ -63,100 +63,64 @@ PARALLEL_BUILDS
 @Library('kernelci') _
 import org.kernelci.util.Job
 
-def buildConfig(kdir, kci_core) {
-    def jopt = ""
-
-    if (params.PARALLEL_BUILDS)
-        jopt = "-j${params.PARALLEL_BUILDS}"
-
-    dir(kci_core) {
-        sh(script: """\
-./kci_build \
-build_kernel \
---kdir=${kdir} \
---defconfig=${params.DEFCONFIG} \
---arch=${params.ARCH} \
---build-env=${params.BUILD_ENVIRONMENT} \
---verbose \
-${jopt} \
-|| echo 'Kernel build failed'
-""")
-
-        sh(script: """\
-./kci_build \
-install_kernel \
---kdir=${kdir} \
---config=${params.BUILD_CONFIG} \
---describe=${params.GIT_DESCRIBE} \
---describe-verbose=${params.GIT_DESCRIBE_VERBOSE} \
---commit=${params.COMMIT_ID} \
-""")
-
-        withCredentials([string(credentialsId: params.KCI_TOKEN_ID,
-                                variable: 'SECRET')]) {
-            sh(script: """\
-./kci_build \
-push_kernel \
---kdir=${kdir} \
---token=${SECRET} \
---api=${params.KCI_API_URL} \
-""")
-
-            sh(script: """\
-./kci_build \
-publish_kernel \
---kdir=${kdir} \
---token=${SECRET} \
---api=${params.KCI_API_URL} \
-""")
-        }
-    }
-
-    dir("linux/_install_") {
-        archiveArtifacts("*.json")
-    }
-}
-
-node("docker" && params.NODE_LABEL) {
+/* K8S build */
+node("docker" && "k8s" && params.NODE_LABEL) {
     def j = new Job()
     def kci_core = "${env.WORKSPACE}/kernelci-core"
-    def kdir = "${env.WORKSPACE}/linux"
-    def docker_image = null
```

Comment:
Please keep this line as the `docker_image` variable is still being used.  It was needed to declare it before because the value was set in the `Init` stage with a reduced scope. Now I know this works without defining the variable but still I think it's better to keep things consistent by declaring all the variables (well, just noticed other ones were never declared...).

Reply:
ok, will add it back